nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentFeature.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.platform.comment;
+
+import org.nuxeo.ecm.platform.test.PlatformFeature;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+/**
+ * Feature for the comment stack and its provides the default implementation of
+ * {@link org.nuxeo.ecm.platform.comment.api.CommentManager}.
```

Comment:
What do you mean? That the feature provides the default implementation? Doesn't it provide _all_ implementations.

Reply:
yes exactly, we have one implementation at a given time: `PropertyCommentManager` or `TreeCommentManager`, even the old one `CommentManagerImp`


actually there is mainly two implementations 

`PropertyCommentManager` -> default one on `10.10`, will deprecated on `master`
`TreeCommentManager` -> provided by `TreeCommentFeature` not the default one because od backward compatibility + need to do migration on `10.10`, and it will be deprecated on `master` because the `CommentFeature` will deploy the `TreeCommentManager` and i will add a new one `PropertyCommentFeature` to keep the tests coverture on the old implementation `PropertyCommentManager`

Reply:
OK so you could say "Feature for the comment stack, which provides the default implementation..."
Would it make sense to add that the default implmentation is `PropertyCommentManager`?